main/dapps/alpha-release-candidate.md

```diff
@@ -0,0 +1,159 @@
+   entity which calls `E(zoe).startInstance(...)`. The creatorFacet
+   can be used to create invitations for other parties, and take actions
+   that are unrelated to making offers.
+2. `creatorInvitation` - a Zoe invitation that is only given to the entity
+   with calls `E(zoe).startInstance(...)`. This invitation is usually
+   used for cases in which a party has to make an offer first, such as
+   escrowing the underlying good for sale in an auction or covered
+   call.
+3. `publicFacet` - an object that will be available through Zoe to anyone who knows
+   the contract instance. The publicFacet is good for general queries,
+   such as getting the current price.
+
+### Access contract terms
+
+To get information such as the issuers, brands, or custom terms that
+a contract was instantiated with, do:
+
+```js
+const { brands, issuers, terms } = zcf.getTerms();
+```
+Note that `brands` and `issuers` are records with keyword keys,
+formerly called `brandKeywordRecord` and `issuerKeywordRecord`,
+respectively.
+
+### Seats (no more offerHandles!)
+
+There are no more offerHandles! Instead, we took a more object-oriented
+approach and created a new `seat` per `E(zoe).offer(...)` call. These seats
+have methods such as `getProposal()`, `getCurrentAllocation()`, and `exit()`. There
+are two important "facets" for seats (a facet is a particular view or API of
+an object; there may be multiple such APIs per object): the ZCFSeat
+and the UserSeat. 
+
+### ZCFSeat Facet 
+
+The ZCFSeat is what is passed to the
+`offerHandlers`, like this one:
+
+```js
+const mintPayment = seat => {
+  const amount = amountMath.make(1000);
+  // Synchronously mint and allocate amount to seat.
+  zcfMint.mintGains({ Token: amount }, seat);
+  // Exit the seat so that the user gets a payout.
+  seat.exit();
+  // Since the user is getting the payout through Zoe, we can
+  // return anything here. Let's return some helpful instructions.
+  return 'Offer completed. You should receive a payment from Zoe';
+};
+```
+
+To learn more about the API of the ZCFSeat, see the type definitions
+[here](https://github.com/Agoric/agoric-sdk/blob/7058a852c46625e28aa9a290b2c99f2a39d0cba5/packages/zoe/src/types.js#L377)
+
+### UserSeat Facet
+
+The UserSeat is what is returned to the caller of
+`E(zoe).offer(invitation, proposal, payments)`. The UserSeat has
+methods such as `getPayout` and `exit`. See the type definitions
+[here](https://github.com/Agoric/agoric-sdk/blob/7058a852c46625e28aa9a290b2c99f2a39d0cba5/packages/zoe/src/types.js#L88)
+
+### Reallocating
+
+To reallocate, first stage an allocation on a seat:
+
+```js
+const creatorStage = creatorSeat.stage({
+    Tip: tipAmountMath.add(creatorTips, tipAmount),
+    Assurance: assuranceMath.getEmpty(),
+  });
+```
+
+Once you have a few seatStagings to reallocate over, you can call
+reallocate like so, with any number of stagings passed in as separate arguments:
```

Comment:
```suggestion
reallocate like so, with at least two stagings passed in as separate arguments:
```

I'd also mention whether they're allowed to include duplicate seats in the set of stagings. (I think the answer is "no".)